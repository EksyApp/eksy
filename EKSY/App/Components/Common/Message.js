import React, {Component} from 'react'
import {Text, StyleSheet, ScrollView} from 'react-native'
import * as Theme from '../../Theme'
import PropTypes from 'prop-types';
import {StyleShape} from "../../Utils/PropTypeShapes";

export class Message extends Component {
	
	render() {
		return(
				<Text {...this.props} style={[style.text, this.props.style]} />
		)
	}
	
}

const style = StyleSheet.create({
	text: {
		textAlign: 'center',
		width: '100%',
		fontSize: 18,
		color: Theme.fontColor
	}
})


Message.propTypes = {
style : StyleShape
}

export default Message
