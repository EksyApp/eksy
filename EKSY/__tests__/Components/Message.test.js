import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Message from "../../App/Components/Common/Message";

test('Message renders correctly', () => {
	const tree = renderer.create(
			<Message />
	).toJSON();
	expect(tree).toMatchSnapshot();
});