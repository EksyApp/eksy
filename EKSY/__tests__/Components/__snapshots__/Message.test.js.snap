// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "rgb(89,85,78)",
        "fontSize": 18,
        "textAlign": "center",
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;
